dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05305AddPushPublishFilterColumnTest.java

```diff
@@ -0,0 +1,46 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import com.dotmarketing.exception.DotDataException;
+import java.sql.SQLException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task05305AddPushPublishFilterColumnTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)